chainerx_cc/chainerx/routines/manipulation.cc

```diff
@@ -695,4 +695,51 @@ std::vector<Array> Split(const Array& ary, std::vector<int64_t> indices, int8_t
     return out;
 }
 
+Array Atleast2D(const Array& x) {
+    Array out;
+    Shape shape = x.shape();
+    Strides strides = x.strides();
+    switch (x.ndim()) {
+        case 0:
+            out = x.Reshape({1, 1});
+            break;
+        case 1:
+            shape.insert(shape.begin(), 1);
```

Comment:
Let's declare `shape` and `strides` after `case 1:`. They're used only in this case.
(Please do not forget to add `{ ... }` around the block)

Reply:
Like this?
```Cpp
case 1: {
    Shape shape = x.shape();
    Strides strides = x.strides();
    shape.insert(shape.begin(), 1);
    strides.insert(strides.begin(), 0);
    out = internal::MakeArray(shape, strides, x.dtype(), x.device(), x.data());
} break;
default:
```